app/routes/task_routes.py

```diff
@@ -1 +1,136 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort,make_response, request, Response, current_app
```

Comment:
Nitpick: inconsistent spacing
```suggestion
from flask import Blueprint, abort, make_response, request, Response, current_app
```